src/Paket.Core/Dependencies/NuGetV3.fs

```diff
@@ -277,7 +282,8 @@ let private getCatalogPageDirectory(basePath:String,item:String) =
         | _ -> Uri.EscapeDataString(item)
     let cachePath = Path.Combine(basePath,hostPath)
     let directory = new DirectoryInfo(cachePath)
-    if directory.Exists |> not then 
+    if directory.Exists |> not then
+        traceWarnfn "Create \"%s\"" directory.FullName
```

Comment:
Is this warning expected?

Reply:
it's not, as the page cache for the only code using this -- the public catalog integration test --
 _should_ be committed (it is now) and the warning will trigger only if called without the path existing